### ISSUE TITLE
Aborts if no `package-lock.json` is found

### ISSUE THREAD

**Issue by Mottie** [silverwind/ver#10]
If a repo has `package-lock=false` set in the `.npmrc` file, `ver` should not use `findUp` to look for a `package-lock.json` file.

If the file is not found, the code will abort:

```
Running "exec:patch" (exec) task
No replacement made in C:\Repos\package-lock.json
>> Exited with code: 1.
>> Error executing child process: Error: Process exited with code 1.
Warning: Task "exec:patch" failed. Use --force to continue.

Aborted due to warnings.
```

In this case, `C:\Repos\package-lock.json` is two parent folders above the repo that is set to not use a package-lock.

**Comment by the-j0k3r** [comment]
Ive added package lock by making it `package-lock=true` set in the `.npmrc`

deleted node_modules and rerun `npm i` it still fails :/

**Comment by Mottie** [comment]
v5.1.4 is working for me:

```
Running "exec:patch" (exec) task
+ git commit -a -m 1.22.115
[master 7c7f5969] 1.22.115
 4 files changed, 15 insertions(+), 6 deletions(-)
+ git tag -f -m 1.22.115 v1.22.115

Done.
```

Let's continue this conversation in Discord instead of here.

**Comment by silverwind** [comment]
Fixed in 5.2.0. It will no longer look outside the directory hosting `package.json`. For `package.json`, it will still potentially go up to the root folder, but there's really no sane way to prevent that.
